src/features/import/components/ImportDialog/Configure/Mapping/MappingRow.tsx

```diff
@@ -21,7 +22,7 @@ import { Msg, useMessages } from 'core/i18n';
 interface MappingRowProps {
   clearConfiguration: () => void;
   column: UIDataColumn<Column>;
-  columnOptions: Option[];
+  columnOptions: Option[][];
```

Comment:
I think we need to change this data structure a little bit, as expressed in another comment. Here are some suggestions:

* Split it up into three separate props of type `Option[]`, e.g. `idOptions`, `fieldOptions` and `otherOptions`
* Change the type of `columnOptions` to be an object with three pre-defined fields, i.e. `{ idOptions: Option[], fieldOptions: Option[], otherOptions: Option[] }`

But all of this makes me think about a wider topic, that I never considered part of the scope for this issue, but am now thinking is an opportunity to improve the code now that we've come across it.

Why are we passing in arrays of hard-coded values to this component, instead of just hard-coding them inside the component? I'm thinking now about the ID field option and the "Other" options. I think they could just be hard-coded inside this component.

For that reason, I think my main proposal is to rename `columnOptions` to `fieldOptions: Option[]` and pass in the fields from outside. For the ID and other options, hard-code them inside this component instead.